mtp_cashbook/apps/cashbook/views.py

```diff
@@ -0,0 +1,23 @@
+from django.contrib.auth.decorators import login_required
+from django.utils.decorators import method_decorator
+from django.views.generic import ListView
+from mtp_cashbook.apps.mtp_auth.api_client import get_connection
+
+
+class TransactionListView(ListView):
```

Comment:
Could we rename this to something which indicates it's a batch of payments rather than all please?